libs/product/driver/magento/src/transforms/product-preview.ts

```diff
@@ -44,6 +47,7 @@ export function transformMagentoSimpleProductPreview(product: MagentoProductPrev
     price: getPrice(product),
     discount: getDiscount(product),
     images: transformMediaGalleryEntries(product, mediaUrl),
+    in_stock: product.stock_status === MagentoProductStockStatusEnum.InStock,
```

Comment:
Should be tested in the preview transform test.

Reply:
done